util/retry.go

```diff
@@ -11,7 +11,7 @@ import (
 // MaxRetriesExceeded error.
 func DoWithRetry(actionDescription string, maxRetries int, sleepBetweenRetries time.Duration, logger *logrus.Entry, action func() error) error {
```

Comment:
This method will need to take in the log level to use for `actionDescription`. No way to know from inside the method otherwise if this should be Info or Debug or whatever else.

Reply:
@brikis98 this is used in two places only: in [remote/remote_state_gcs.go#L272](https://github.com/amnk/terragrunt/blob/amnk/change-default-log-level/remote/remote_state_gcs.go#L272) and [remote/remote_state_s3.go#L364](https://github.com/amnk/terragrunt/blob/amnk/change-default-log-level/remote/remote_state_s3.go#L364). 

I have changes the method to take logLevel as input, and configured both places as `Debugf`. Hope that's ok

Reply:
Looks great, thanks!